core-bundle/src/Altcha/Altcha.php

```diff
@@ -78,6 +72,13 @@ public function validate(string $payload): bool
             return false;
         }
 
+        parse_str(explode('?', $json['salt'], 2)[1] ?? '', $parameters);
+        $expiry = (int) ($parameters['expires'] ?? 0);
+
+        if ($expiry < time()) {
+            throw new ChallengeExpiredException('The given challange has expired. Please try again.');
```

Comment:
Should we just return false here instead?

Reply:
Yes. Changed in 7738007db243a4cb9ad30ecf61e41226faae1809.